source/blog/2016-07-20-vagrant-service-manager-install-cli.html.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Download client binaries through vagrant-service-manager in ADB
+author: budhram
+date: 2016-07-20 09:56 UTC
+tags: atomic-developer-bundle, vagrant-service-manager
+published: true
+comments: true
+---
+
+The [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) plugin is designed to enable easier access to the features and services provided by the [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle). More information can be found in the README  of 'vagrant-service-manager' repo.
+
+This blog is about the new feature in 'vagrant-service-manger' i.e. `install-cli` command. `install-cli` got released with 'vagrant-service-manager' version 1.2.0. This command installs the client binary for any of the services provided by ADB. At this point it can download client binaries for docker and OpenShift.
```

Comment:
As part of the effort to continually improve the developer experience and make getting started easier, the ADB now supports client binary downloads.  These downloads are facilitated by a new feature in 'vagrant-service-manger', the `install-cli` command.

The [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) plugin enables easier access to the features and services provided by the [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle). More information can be found in the README  of 'vagrant-service-manager' repo.

The `install-cli` command was released as part of 'vagrant-service-manager' version 1.2.0. This command installs the client binary for services provided by the ADB. Today it can download client binaries for docker and OpenShift.  This feature allows developers to know they have the best client for use with the ADB services they are using.